.github/workflows/tests.yml

```diff
@@ -33,6 +33,28 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        # cpu version of pytorch
+        pip install .[test]
+        pip install git+https://github.com/huggingface/peft.git@main
+    - name: Test with pytest
+      run: |
+        make test
+  tests_no_peft:
```

Comment:
Why is this step necessary? If the tests run with peft they should also run without, no?

Reply:
Removed !